g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v2/formatting/pattern/FormattingPatternAPI.java

```diff
@@ -88,10 +90,26 @@ public APIResponseDTO getI18nPatternWithLanguageAndRegion(
         }
 
         Map<String, Object> patternMap = patternService.getPatternWithLanguageAndRegion(language, region, categories, scopeFilter);
-        if (!(boolean)patternMap.get("isExistPattern")) {
-            return super.handleResponse(APIResponseStatus.INTERNAL_NO_RESOURCE_ERROR, "'No mapping language found for region!");
+
+        int emptyCount = getEmptyCategoryCount(patternMap, categories);
+        if(emptyCount == 0) {
```

Comment:
If there are no category, emptyCount equals 0 as well.

Reply:
‘scope’ parameter is required, so no category scenario won't appear.